cli/src/main.rs

```diff
@@ -75,7 +75,7 @@ fn run<I: IntoIterator<Item = T>, T: Into<OsString> + Clone>(args: I) -> Result<
                     Arg::with_name("target_rate")
                         .long("target-rate")
                         .takes_value(true)
-                        .required(true)
+                        .default_value("1/s")
```

Comment:
Usually this is done as a `get_value(...).unwrap_or_else("...")`

Reply:
Ah, I see it's already there but it was being blocked by the required(true) clap option. I've updated the PR to just remove this line.